t/t7527-internal-fsmonitor.sh

```diff
@@ -171,7 +171,7 @@ test_expect_success 'setup' '
 	test_tick &&
 	git -c core.fsmonitor= commit -m initial &&
 
-	git config core.fsmonitor :internal:
+	git config feature.fsmonitor true
```

Comment:
There are a lot of lines in this file that do `git config --unset core.fsmonitor` to undo the `:internal:` setting.  We need to update them too.

Reply:
Hmm. Maybe we should call it `feature.internalFSMonitor`, then?

Reply:
I went for `core.useBuiltinFSMonitor`, after all.